params/config.go

```diff
@@ -765,12 +765,12 @@ func (c *ChainConfig) CheckConfigForkOrder() error {
 		{name: "mirrorSyncBlock", block: c.MirrorSyncBlock},
 		{name: "brunoBlock", block: c.BrunoBlock},
 		{name: "eulerBlock", block: c.EulerBlock},
+		{name: "bonehBlock", block: c.BonehBlock},
+		{name: "lynnBlock", block: c.LynnBlock},
 		{name: "berlinBlock", block: c.BerlinBlock},
```

Comment:
The following four lines can be deleted. That's what we do in develop branch

Reply:
Cool, I think we can rebase develop branch then.